acl/testing.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 )
 
-func RequirePermissionDeniedError(t testing.TB, err error, _ Authorizer, _ *AuthorizerContext, resource Resource, accessLevel AccessLevel, resourceID string) {
+func RequirePermissionDeniedError(t testing.TB, err error, authz *Authorizer, _ *AuthorizerContext, resource Resource, accessLevel AccessLevel, resourceID string) {
```

Comment:
Is `Authorizer` an interface here? I don't think it should be a pointer to the interface if that's the case.